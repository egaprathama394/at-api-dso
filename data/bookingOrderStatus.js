export default {
  orderDaimoCode: 'DFA202000000123',
  pssOrderStatus: '2',
};
