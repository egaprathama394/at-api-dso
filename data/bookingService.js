const randomstring = require('randomstring');

const fullNameMax = randomstring.generate({
  charset: 'string',
  length: '110',
});

const noPhoneMax = randomstring.generate({
  charset: 'numeric',
  length: '17',
});

const noPhoneRandom = randomstring.generate({
  charset: 'numeric',
  length: '18',
});

const fullNameRandom = randomstring.generate({
  charset: 'string',
  length: '111',
});

const randomNopol1 = randomstring.generate({
  charset: 'numeric',
  length: '4',
});

const randomNopol2 = randomstring.generate({
  charset: 'numeric',
  length: '4',
});

const randomNopol3 = randomstring.generate({
  charset: 'numeric',
  length: '4',
});

const randomNopol4 = randomstring.generate({
  charset: 'numeric',
  length: '4',
});

export default {
  address: '',
  bookingDateTime: '2020-04-01 10:00:00',
  branchCode: 'D004',
  city: 'DKI JAKARTA',
  fullName: '',
  jobService: [
    {
      actual: 0.0,
      id: 0,
      isActive: false,
      isPickedJob: false,
      jobCode: '10000',
      jobDescription: 'CHECKING 10.000',
      price: 0,
      rate: 0,
    },
  ],
  notes: 'Data submit booking service',
  orderEndTime: '2020-04-01 15:00:00',
  orderStartTime: '2020-04-01 10:00:00',
  phoneNo: '',
  policeRegNo: `AB${randomNopol1}XYZ`,
  vehicleType: 'LUXIO',
  vin: '',
  year: 2014,
  AccountType: '1',
  Action: '',
  ApplicationName: '8EA6E777-AA05-4CAF-9F54-5B6A106760E2',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

// data for submit booking service max lenght

export const maxInput = {
  address: '',
  bookingDateTime: '2020-04-01 10:00:00',
  branchCode: 'D004',
  city: 'DKI JAKARTA',
  fullName: `${fullNameMax}`,
  jobService: [
    {
      actual: 0.0,
      id: 0,
      isActive: false,
      isPickedJob: false,
      jobCode: '10000',
      jobDescription: 'CHECKING 10.000',
      price: 0,
      rate: 0,
    },
  ],
  notes: 'Data booking service dengan inputan max char',
  orderEndTime: '2020-04-01 15:00:00',
  orderStartTime: '2020-04-01 10:00:00',
  phoneNo: `081${noPhoneMax}`,
  policeRegNo: `AC${randomNopol2}XYZ`,
  vehicleType: 'LUXIO',
  vin: '',
  year: 2014,
  AccountType: '1',
  Action: '',
  ApplicationName: '8EA6E777-AA05-4CAF-9F54-5B6A106760E2',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

// data for submit booking service max lenght + 1

export const negativeRequest = {
  address: '',
  bookingDateTime: '2020-04-01 10:00:00',
  branchCode: 'D004',
  city: 'DKI JAKARTA',
  fullName: `${fullNameRandom}`,
  jobService: [
    {
      actual: 0.0,
      id: 0,
      isActive: false,
      isPickedJob: false,
      jobCode: '10000',
      jobDescription: 'CHECKING 10.000',
      price: 0,
      rate: 0,
    },
  ],
  notes: 'Data submit booking service dengan inputan max + 1 char',
  orderEndTime: '2020-04-01 15:00:00',
  orderStartTime: '2020-04-01 10:00:00',
  phoneNo: `081${noPhoneRandom}`,
  policeRegNo: `AD${randomNopol3}XYZ`,
  vehicleType: 'LUXIO',
  vin: '',
  year: 2014,
  AccountType: '1',
  Action: '',
  ApplicationName: '8EA6E777-AA05-4CAF-9F54-5B6A106760E2',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

// data for cancel booking service

export const inputDataForCancel = {
  address: '',
  bookingDateTime: '2020-04-01 10:00:00',
  branchCode: 'D004',
  city: 'DKI JAKARTA',
  fullName: '',
  jobService: [
    {
      actual: 0.0,
      id: 0,
      isActive: false,
      isPickedJob: false,
      jobCode: '10000',
      jobDescription: 'CHECKING 10.000',
      price: 0,
      rate: 0,
    },
  ],
  notes: 'Data untuk cancel booking service',
  orderEndTime: '2020-04-01 15:00:00',
  orderStartTime: '2020-04-01 10:00:00',
  phoneNo: '',
  policeRegNo: `AA${randomNopol4}XYZ`,
  vehicleType: 'LUXIO',
  vin: '',
  year: 2014,
  AccountType: '1',
  Action: '',
  ApplicationName: '8EA6E777-AA05-4CAF-9F54-5B6A106760E2',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

export const cancelRequest = {
  orderNo: '',
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
};

// data for display booking service

export const displayBookingService = {
  isWalkIn: false,
  orderBy: '',
  AccountType: '1',
  Action: '',
  ApplicationName: '8EA6E777-AA05-4CAF-9F54-5B6A106760E2',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

export const invalidDisplayBookingService = {
  isWalkIn: '',
  orderBy: '',
  AccountType: '',
  Action: '',
  ApplicationName: '',
  DeviceType: '',
  OperatingSystem: '',
  UserVersion: '',
};
