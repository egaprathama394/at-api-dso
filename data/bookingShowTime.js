export default {
  branchCode: 'D004',
  scheduleDate: '2020-09-30 00:00:00',
  vehicleType: 'XENIA MT',
  jobs: [
    {
      jobCode: 'OTHER',
      jobDescription: 'PERBAIKAN LAINNYA',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
};

export const negativeRequest = {
  branchCode: 'D004',
  scheduleDate: '2018-10-26 00:00:00',
  vehicleType: 'XENIA MT',
  jobs: [
    {
      jobCode: 'OTHER',
      jobDescription: 'PERBAIKAN LAINNYA',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
};
