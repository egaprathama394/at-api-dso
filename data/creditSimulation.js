export default {
  P_NEW_USED: 'N',
  P_CATEGORY: 'EX2',
  P_VEHICLE_CATEGORY: '1',
  P_BRAND: '002',
  P_TYPE: '1',
  P_MODEL: '1',
  P_DP: '49125000',
  P_OTR: '196500000',
  P_TYPE_INSU: 'A',
  P_MODE_INSU: 'K',
  P_FLAG_ACP: 'Y',
  P_TENORVALUE: '12',
};