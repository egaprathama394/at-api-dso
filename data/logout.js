export default {
  applicationName: '8EA6E777-AA05-4CAF-9F54-5B6A106760E2',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
  deviceID: '8B55A4DA-5331-47F9-9C8D-838A110FA18C',
};


export const negativeRequest = {
  action: 'Display',
  applicationName: '9093D285-5263-4B9A-97D7-C185189B0695',
  userVersion: '1.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
};
