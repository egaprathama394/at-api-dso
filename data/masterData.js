
export default {
  companyCode: '0003',
  masterDataInfo: [
    {
      lastSync: '1900-01-01 00:00:00',
      objectName: 'Branch',
    },
  ],
  vehicleBrandCode: 'DAIHATSU',
  accountType: '1',
  action: '',
  deviceType: '0',
  operatingSystem: '0',
  serVersion: '1.0.0',
};

export const JobServiceCompany = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'JobService_Company',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',

};

export const VehicleBrand = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'vehicleBrand',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
};

export const district = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'district',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',

};

export const province = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'province',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',

};

export const region = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'region',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',

};

export const holidayCalendar = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'holidayCalendar',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',

};

// looping data for job service company

export const loopVehicleType = [
  {
    vehicleType: 'AYLA',
    title: 'test case get master data job service company where vehicle = AYLA',
  },
  {
    vehicleType: 'GRANMAX',
    title: 'test case get master data job service company where vehicle = GRANMAX',
  },
  {
    vehicleType: 'GRANMAX1.3',
    title: 'test case get master data job service company where vehicle = GRANMAX1.3',
  },
  {
    vehicleType: 'GRANMAX1.5',
    title: 'test case get master data job service company where vehicle = GRANMAX1.5',
  },
  {
    vehicleType: 'LUXIO',
    title: 'test case get master data job service company where vehicle = LUXIO',
  },
  {
    vehicleType: 'Sigra 1.0',
    title: 'test case get master data job service company where vehicle = Sigra 1.0',
  },
  {
    vehicleType: 'Sigra 1.2',
    title: 'test case get master data job service company where vehicle = Sigra 1.2',
  },
  {
    vehicleType: 'SIRION',
    title: 'test case get master data job service company where vehicle = SIRION',
  },
  {
    vehicleType: 'TERIOS',
    title: 'test case get master data job service company where vehicle = TERIOS',
  },
  {
    vehicleType: 'XENIA1.0',
    title: 'test case get master data job service company where vehicle = XENIA1.0',
  },
  {
    vehicleType: 'XENIA1.3',
    title: 'test case get master data job service company where vehicle = XENIA1.3',
  },
  {
    vehicleType: 'OTHERS',
    title: 'test case get master data job service company where vehicle = OTHERS',
  },
];

export const VehicleTestDrive = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'VehicleTestDrive',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
};

export const parameterConfigApp = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'ParameterConfigApplication',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
};
