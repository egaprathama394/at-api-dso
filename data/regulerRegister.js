const randomstring = require('randomstring');

const randomMaxEmail = randomstring.generate({
  charset: 'string',
  length: '81',
});

const randomMaxfirstName = randomstring.generate({
  charset: 'string',
  length: '50',
});

const randomMaxlastName = randomstring.generate({
  charset: 'string',
  length: '50',
});

export default {
  Email: 'renni.aja8@gmail.com',
  FirstName: 'test',
  LastName: 'registrasi',
  Password: 'o02k6B5cS+ULS+KuYm2LtSNw9fNJbHfjdQs6VJ5hY7p+Onf9gf6uyKNwT4hckRzTDEmglwTOJE1y\n9dwzhShshdEqBfCWoixMCHS497qFqWp21IZcWmZ8b9+EtoUeFBrH3byP+oVH0Y405HvofHmAr6HK\nQhccv5BUY7oEsyYVs526dOE6adJzpM2kV0R+P+FU7UUWtDhgjbmhJwNFP54hbBFquvXhaeJuNxFe\n9jWFZ5qk3FIP7bieILDSIkRGdHaYdJF4xtYMXodG4VQiMIV8DTAU6CFIjTMQfM7+PTJZqcxthcy3\nv367FNtCFaS/H/MQAuB/Q5MtlOdjpENevqWW2w\u003d\u003d\n',
  PhoneNo: '081456155090',
  AccountType: '1',
  Action: '',
  ApplicationName: '35AE57AD-572F-4EAF-B908-3B0523ADEDB8',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

export const regsiteredEmailRequest = {
  Email: 'testerlogin.DSO@gmail.com',
  FirstName: 'test',
  LastName: 'registrasi',
  Password: '0S8AWFvrTbTXXHGFTOZS/7s22ZrJLFQqjKqFDUyewpQFslyF5xbwEONGTrRIjLzTCHk7EVYKZAm0\nv+mu9QxLRwvXJ4JYqE5ElGB1UmFhrhKriW4aGp6dJI1PUsnuokMzP4I1v1CH3DLBh4TtsFwHMc6v\n+MVOVp3ogspKpI7xupnDnt+YmNVLOsjq9bh9GFy9g5mXwRwReMWAhL1RLiLTK/G8rY4SSWhb/WFZ\nIPXhYhIXfUpkIzaK9UPN05da0qbe6pjvSIUzUdSgxkIr6DEM/D45W7VrlyFAJ4WkHVPtVjX0Yrii\nXnd/Lf0KuaYNbX1zD3/K+ym0MuyRnOfPAQ9geg\u003d\u003d\n',
  PhoneNo: '081456155090',
  AccountType: '1',
  Action: '',
  ApplicationName: '35AE57AD-572F-4EAF-B908-3B0523ADEDB8',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

export const inactiveEmail = {
  Email: 'xxxyyy0990@gmail.com',
  FirstName: 'test',
  LastName: 'registrasi',
  Password: 'HOek+lGLDkj9P8dGmpDfzvfRmGUvd3qwM+YpWhf1d68qw+/CJwNGDEO/WLu9vMT0YVJdipIprAYX\nRT9YWir6VMg98JYRnjyw3E72AemTr+xhvj37Sdb6KRZimOVlumibwDxSACXgHtr8XIj/MdeFnTEF\nezgsv2u2o5EC6GXQFrUkPJq0cHgoNNrPkUKMY2k35C//APKBiUX1YDFHzBsX77DqU3DSAA8S3ACn\ngsN0UvhLPJD3+xw5aWJvUQx/PrCPMOMjKOPXwd13dkFWK47UMOGcDxWGoYPbEwz17WV5WNijt+KC\n6wKC52JZiwJeqAOrUNHzb+lgT3mon/65DA3J+w\u003d\u003d\n',
  PhoneNo: '081456155090',
  AccountType: '1',
  Action: '',
  ApplicationName: '35AE57AD-572F-4EAF-B908-3B0523ADEDB8',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

// data forregular registrasi input max length

export const maxlengthInput = {
  Email: 'testingAT.ABCDEFGHIJABCDEFGHIJABCDEFGHIJABCDEFGHIJABCDEFGHIJABCDEFGHIJABCDEFGHIJABCDEFGHIJ@gmail.com',
  FirstName: `${randomMaxfirstName}`,
  LastName: `${randomMaxlastName}`,
  Password: 'HOek+lGLDkj9P8dGmpDfzvfRmGUvd3qwM+YpWhf1d68qw+/CJwNGDEO/WLu9vMT0YVJdipIprAYX\nRT9YWir6VMg98JYRnjyw3E72AemTr+xhvj37Sdb6KRZimOVlumibwDxSACXgHtr8XIj/MdeFnTEF\nezgsv2u2o5EC6GXQFrUkPJq0cHgoNNrPkUKMY2k35C//APKBiUX1YDFHzBsX77DqU3DSAA8S3ACn\ngsN0UvhLPJD3+xw5aWJvUQx/PrCPMOMjKOPXwd13dkFWK47UMOGcDxWGoYPbEwz17WV5WNijt+KC\n6wKC52JZiwJeqAOrUNHzb+lgT3mon/65DA3J+w\u003d\u003d\n',
  PhoneNo: '081456155090',
  AccountType: '1',
  Action: '',
  ApplicationName: '35AE57AD-572F-4EAF-B908-3B0523ADEDB8',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

// data forregular registrasi input max length +1 chars

export const negativeRequest = {
  Email: `testingAT.${randomMaxEmail}@gmail.com`,
  FirstName: `${randomMaxfirstName}`,
  LastName: `${randomMaxlastName}`,
  Password: 'HOek+lGLDkj9P8dGmpDfzvfRmGUvd3qwM+YpWhf1d68qw+/CJwNGDEO/WLu9vMT0YVJdipIprAYX\nRT9YWir6VMg98JYRnjyw3E72AemTr+xhvj37Sdb6KRZimOVlumibwDxSACXgHtr8XIj/MdeFnTEF\nezgsv2u2o5EC6GXQFrUkPJq0cHgoNNrPkUKMY2k35C//APKBiUX1YDFHzBsX77DqU3DSAA8S3ACn\ngsN0UvhLPJD3+xw5aWJvUQx/PrCPMOMjKOPXwd13dkFWK47UMOGcDxWGoYPbEwz17WV5WNijt+KC\n6wKC52JZiwJeqAOrUNHzb+lgT3mon/65DA3J+w\u003d\u003d\n',
  PhoneNo: '081456155090',
  AccountType: '1',
  Action: '',
  ApplicationName: '35AE57AD-572F-4EAF-B908-3B0523ADEDB8',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};
