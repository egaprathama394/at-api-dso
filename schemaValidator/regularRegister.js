export default {
  properties: {
    acknowledge: {
      type: 'integer',
    },
    message: {
      type: 'string',
    },
    version: {
      type: 'string',
    },
    build: {
      type: 'string',
    },
    errorCode: {
      type: 'integer',
    },
  },
  required: [
    'acknowledge',
    'message',
    'version',
    'build',
    'errorCode',
  ],
  additionalProperties: false,
};

export const negativeRequestSchema = {
  properties: {
    message: {
      type: 'string',
    },
  },
  required: [
    'message',
  ],
  additionalProperties: false,
};
