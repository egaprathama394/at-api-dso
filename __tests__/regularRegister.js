import postRegist from '../endpoints/regularRegister';
import * as registSchema from '../schemaValidator/regularRegister';
import * as body from '../data/regulerRegister';

const { matchers } = require('jest-json-schema');

expect.extend(matchers);

describe('Regular Registration', () => {
  test('Can registered using unregistered email - positive test cases', async () => {
    const response = await postRegist(body.default);
    expect(response.status).toEqual(200);
    expect(response.body).toMatchSchema(registSchema.default);
    Reflect.deleteProperty(response.body, 'build');
    expect(response.body).toMatchSnapshot();
  });

  test('Can registered using input max at email, first name, and last name - positive test cases', async () => {
    const response = await postRegist(body.maxlengthInput);
    expect(response.status).toEqual(200);
    expect(response.body).toMatchSchema(registSchema.default);
    Reflect.deleteProperty(response.body, 'build');
    expect(response.body).toMatchSnapshot();
  });

  test('Can not registration using registered email - negative test cases', async () => {
    const response = await postRegist(body.regsiteredEmailRequest);
    expect(response.status).toEqual(200);
    expect(response.body).toMatchSchema(registSchema.default);
    Reflect.deleteProperty(response.body, 'build');
    expect(response.body).toMatchSnapshot();
  });

  test('Can not registration using email register but not active yet - negative test cases', async () => {
    const response = await postRegist(body.inactiveEmail);
    expect(response.status).toEqual(200);
    expect(response.body).toMatchSchema(registSchema.default);
    Reflect.deleteProperty(response.body, 'build');
    expect(response.body).toMatchSnapshot();
  });

  test('Can not registration using input max+1 chars at email, first name, and last name - negative test cases', async () => {
    const response = await postRegist(body.negativeRequest);
    expect(response.status).toEqual(500);
    expect(response.body).toMatchSchema(registSchema.negativeRequestSchema);
    expect(response.body).toMatchSnapshot();
  });
});
