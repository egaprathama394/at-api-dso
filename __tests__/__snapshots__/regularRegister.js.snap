// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Regular Registration Can not registration using email register but not active yet - negative test cases 1`] = `
Object {
  "acknowledge": 1,
  "errorCode": 0,
  "message": "Account xxxyyy0990@gmail.com is not active. Activation email will be sent. Please check your inbox or spam email for activation.",
  "version": "1.0",
}
`;

exports[`Regular Registration Can not registration using input max+1 chars at email, first name, and last name - negative test cases 1`] = `
Object {
  "message": "An error has occurred. There's problem in database, please try again later.",
}
`;

exports[`Regular Registration Can not registration using registered email - negative test cases 1`] = `
Object {
  "acknowledge": 0,
  "errorCode": 0,
  "message": "Account is already exist. Please use login menu.",
  "version": "1.0",
}
`;

exports[`Regular Registration Can registered using input max at email, first name, and last name - positive test cases 1`] = `
Object {
  "acknowledge": 1,
  "errorCode": 0,
  "message": "SignUp for testingAT.ABCDEFGHIJABCDEFGHIJABCDEFGHIJABCDEFGHIJABCDEFGHIJABCDEFGHIJABCDEFGHIJABCDEFGHIJ@gmail.com is success. Please check your inbox or spam email for activation.",
  "version": "1.0",
}
`;

exports[`Regular Registration Can registered using unregistered email - positive test cases 1`] = `
Object {
  "acknowledge": 1,
  "errorCode": 0,
  "message": "SignUp for renni.aja8@gmail.com is success. Please check your inbox or spam email for activation.",
  "version": "1.0",
}
`;
